### DISCUSSION TITLE
Completely overriding Haxeflixel Executable Icon

### DISCUSSION THREAD

**Discussion by Raltyro** [HaxeFlixel/flixel#2758] [Q&A]
Is there a way to 100% override haxeflixel default exe icon with only one image?
or you just have to do multiples of "<icon/>" with different sizes?

or atleast, how to forcing the executable icon to use a different existing ico file

**Comment by DarDevv** [comment]
add the following code in Project.xml:
`<icon path="assets/path/to/your/image.png" />`
